Engine/Program.cs

```diff
@@ -0,0 +1,239 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using Engine.CliHelpers;
+using Engine.Helpers;
+
+namespace Engine;
+
+public static class Program
```

Comment:
I like to think of the program as being the most basic entrypoint to your code, anything related to bots  or even game flow I would probably try to move out and make this as slim as possible